plugins/modules/k8s_cp.py

```diff
@@ -573,32 +192,17 @@ def execute_module(module):
             msg="Pod contains more than 1 container, option 'container' should be set"
         )
 
-    try:
-        load_class = {"to_pod": K8SCopyToPod, "from_pod": K8SCopyFromPod}
-        state = module.params.get("state")
-        k8s_copy = load_class.get(state)(module, k8s_ansible_mixin.client)
-        k8s_copy.run()
-    except Exception as e:
-        module.fail_json("Failed to copy object due to: {0}".format(to_native(e)))
+    state = module.params.get("state")
+    if state == "to_pod":
+        k8s_copy = K8SCopyToPod(module, k8s_ansible_mixin.client)
+    else:
+        k8s_copy = K8SCopyFromPod(module, k8s_ansible_mixin.client)
+    k8s_copy.run()
```

Comment:
Should we wrap this `run()` call in a try/except block?

Reply:
yes indeed, done